### DISCUSSION TITLE
¿Which is the best way to organize my blog that has standard posts and a section with books?

### DISCUSSION THREAD

**Discussion by ariznaf** [quarto-dev/quarto-cli#8062] [Q&A]
### Description

I have recently discovered quarto. I have been trying to use markdown to create my blog and learning site with jekyll and other static content creators, but I think quarto outperforms all, with its integration of python and R.

My site would be a blog with posts about my activities and publications, but woul have a learning section that would content mainly books about geotechnics (rockmechanics and

The navigation structure I woulld like it to be something like:

/index.qmd (home page with listings of recent posts and books)
/learning/   (the section about learning geotechnics books, I have created a project to content them)
/learning/rockmechanics/ (book about rock mechanics,  I have created a book project for it)
/learning/otherbook/
...

And so on.

I have tried to replicate that structure as a directory tree in my quarto source folder.

But when I render the site, the book folder is also rendered and the book does not work correctly.
I have to render the book folder after that in order to get the book correctly rendered with its chapters.

So it seems that the books should reside in a separate folder structure. It would be better to have a separate github project for each book too.

But I have two problems:

**Comment by mcanouil** [comment]
Your post seems incomplete.
Regarding books inside website, the question has already been asked:
- #3505
Take a look at https://quarto.org/docs/gallery/#websites and https://github.com/mcanouil/awesome-quarto#websites-formats
